protected/views/site/guide.php

```diff
@@ -0,0 +1,465 @@
+                    <ul class="nav nav-tabs nav-border-tabs" style="margin-top: 1px; margin-bottom: 1px">
+                            <li class="active"><a href="/site/guide">General Submission Guidelines</a></li>
+                            <li class="dropdown">
+                                <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
+                                    Datasets Checklists&nbsp;<i class="fa fa-angle-down"></i>
+                                </a>
+                                <ul class="dropdown-menu" style="margin-top: 5px;">
+                                    <li><a href="/site/guidegenomic">Genomic Dataset Checklist</a></li>
+                                    <li><a href="/site/guideimaging">Imaging Dataset Checklist</a></li>
+                                    <li><a href="/site/guidemetabolomic">Metabolomic and Lipidomic Dataset Checklist</a></li>
+                                    <li><a href="/site/guideepigenomic">Epigenomic Dataset Checklist</a></li> 
+                                    <li><a href="/site/guidemetagenomic">Metagenomic Dataset Checklist</a></li>
+                                </ul>
+                            </li>
+                </div>
+            </section>
+            <section>
+                <div class="tab-content">
+                    <div role="tabpanel" class="tab-pane active" id="general">
+                        <h4 style="color: #099242; margin-bottom: 10px;">General Submission Guidelines </h4>
+                        <div class="subsection">
+                            <p>GigaDB is a <a target="_blank" href="https://www.cngb.org/aboutUs.html?i18nlang=en_US">China National GeneBank</a> supported repository used to host data and tools associated with articles in <i>GigaScience</i>. As part of your manuscript submission and in line with the <a target="_blank" href="https://academic.oup.com/gigascience/pages/editorial_policies_and_reporting_standards">Reporting Standards</a> and <a target="_blank" href="http://doi.org/10.25504/fairsharing.prdtva">FAIRsharing guidelines for data deposition and formatting for papers submitted to <i>GigaScience</i></a> we will provide an associated GigaDB dataset to host the data and files required for transparency and reproducibility. GigaDB is an open-access database. As such, all data submitted to GigaDB must be fully consented for public release (for more information about our data policies, please see our <a href="/site/term">Terms of use</a> page).
+                            </p>
+                        </div>      
+                    
+                        <h4 style="color: #099242; margin-bottom: 10px;">Workflow</h4>
+                        <div class="subsection">
+                            <p>The workflow diagram below details a standard submission process:</p>
+                            <img src="/images/workflow.png" alt="Workflow"></img>
```

Comment:
@kencho51 Can you check this `workflow.png` image is visible on your local gigadb?

Reply:
@pli888 The `workflow.png` is not showing in my local gigadb, I think it is because it is missing in the `/image` folder.

Reply:
You can get the image at http://gigadb.org/site/guide and commit the file into the `images` directory so it is part of this PR.